spec/mailers/previews/welcome_email_preview.rb

```diff
@@ -0,0 +1,12 @@
+# Preview all emails at http://localhost:3000/rails/mailers/welcome_email
```

Comment:
Except that I changed the unsubscribe message at the bottom and added a fake address...but the rest is current.